### ISSUE TITLE
Install and Usage code for js-libp2p-webrtc-star

### ISSUE THREAD

**Issue by Annamarie2019** [libp2p/js-libp2p-webrtc-star#431]
For [js-libp2p-webrtc-star-signalling-server](https://github.com/libp2p/js-libp2p-webrtc-star/tree/master/packages/webrtc-star-signalling-server), there are 3 issues:

- The code, `> npm install -g @libp2p/webrtc-star-signalling-server' copies the > sign, so you'll get a message about Usage untill the user realizes the > got copied and removes it. Better to not include the >
- Once you realize the problem and run the code, you get a permissions error. (see image)
- The code,`$ webrtc-star --port=13579 --host=127.0.0.1` ... same problem as above: copies the $

**Comment by johnnymatthews** [comment]
Those instructions leave out the fact that you've got to clone the repository, `cd` into the correct folder, and then run those commands (without the `>` and `$`).

Here's what I did (`>` denotes the output from each command):



1. Clone the repo:

    ```shell
    git clone https://github.com/libp2p/js-libp2p-webrtc-star.git


    > Cloning into 'js-libp2p-webrtc-star'...
    > remote: Enumerating objects: 2665, done.
    > remote: Counting objects: 100% (629/629), done.
    > remote: Compressing objects: 100% (257/257), done.

    > remote: Total 2665 (delta 420), reused 542 (delta 361), pack-reused 2036
    > Receiving objects: 100% (2665/2665), 1.66 MiB | 9.42 MiB/s, done.
    > Resolving deltas: 100% (1617/1617), done.
    ```

2. Move into the directory:

    ```shell
    cd js-libp2p-webrtc-star
    ```

3. Install the local `@libp2p` package:

    ```shell
    npm nstall -g @libp2p/webrtc-star-signalling-server


    > added 82 packages, and audited 83 packages in 7s
    > 
    > 2 packages are looking for funding
    >   run `npm fund` for details
    > 
    > found 0 vulnerabilities
    ```

4. Start the webrtc server:

    ```shell
    webrtc-star --port=13579 --host=127.0.0.1

    > @libp2p/webrtc-star-signalling-server@1.0.5
    > Listening on: http://127.0.0.1:13579
    ```

**Comment by Annamarie2019** [comment]
Thanks, @johnnymatthews  I was able to execute 1 and 2, but I got a permissions error again on 3. (Also, please note typo in "nstall")
<img width="911" alt="Error1" src="https://user-images.githubusercontent.com/46230706/168954947-e06b1502-2a21-4fe3-973a-4f236a3a25b5.png">
<img width="633" alt="Logs 2022-05-17 at 9 08 18 PM" src="https://user-images.githubusercontent.com/46230706/168955424-0f8ce3d6-2497-405d-bff8-8011722fac35.png">

I think I need permissions to libp2p.

**Comment by johnnymatthews** [comment]
No, this isn't a libp2p issue. It looks like your NPM installation has some funky issues going on. I'd recommend completely uninstalling NPM, and then installing [NVM](https://github.com/nvm-sh/nvm), and managing your NPM installations that way.

**Comment by wemeetagain** [comment]
> Those instructions leave out the fact that you've got to clone the repository, `cd` into the correct folder, and then run those commands (without the `>` and `$`).
> 
> Here's what I did (`>` denotes the output from each command):
> 
>     1. Clone the repo:
>        ```shell
>        git clone https://github.com/libp2p/js-libp2p-webrtc-star.git
>        
>        
>        > Cloning into 'js-libp2p-webrtc-star'...
>        > remote: Enumerating objects: 2665, done.
>        > remote: Counting objects: 100% (629/629), done.
>        > remote: Compressing objects: 100% (257/257), done.
>        
>        > remote: Total 2665 (delta 420), reused 542 (delta 361), pack-reused 2036
>        > Receiving objects: 100% (2665/2665), 1.66 MiB | 9.42 MiB/s, done.
>        > Resolving deltas: 100% (1617/1617), done.
>        ```
> 
>     2. Move into the directory:
>        ```shell
>        cd js-libp2p-webrtc-star
>        ```
> 
>     3. Install the local `@libp2p` package:
>        ```shell
>        npm nstall -g @libp2p/webrtc-star-signalling-server
>        
>        
>        > added 82 packages, and audited 83 packages in 7s
>        > 
>        > 2 packages are looking for funding
>        >   run `npm fund` for details
>        > 
>        > found 0 vulnerabilities
>        ```
> 
>     4. Start the webrtc server:
>        ```shell
>        webrtc-star --port=13579 --host=127.0.0.1
>        
>        > @libp2p/webrtc-star-signalling-server@1.0.5
>        > Listening on: http://127.0.0.1:13579
>        ```

Hi @johnnymatthews , thanks for finding shortcomings in the docs
would you be willing to make  a PR that makes the instructions in the README better? :pray: :pray:

**Comment by johnnymatthews** [comment]
@wemeetagain, for sure! Here's a PR: https://github.com/libp2p/js-libp2p-webrtc-star/pull/435
